src/main/java/io/streamnative/pulsar/manager/service/impl/BookiesServiceImpl.java

```diff
@@ -2,9 +2,9 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
```

Comment:
It seems that there are two more p tags, which will cause the licensing head to fail the inspection.
You can use the command to format licensing:
```
./gradlew licenseFormatNode
```

Reply:
@tuteng Addressed the comments